jvector-examples/src/main/java/io/github/jbellis/jvector/example/util/DataSet.java

```diff
@@ -28,7 +28,9 @@ public class DataSet {
     public final List<float[]> queryVectors;
     public final List<? extends Set<Integer>> groundTruth;
 
-    public DataSet(String name, VectorSimilarityFunction similarityFunction, List<float[]> baseVectors, List<float[]> queryVectors, List<? extends Set<Integer>> groundTruth) {
+    public final String details;
```

Comment:
i don't love jamming the parsing information into DataSet, what was wrong with printing it when it's loaded?

Reply:
It's more of stats info (ideally, if we need it in different format).
It is better to have an option to construct output later/as needed, in the format that suitable for parsers instead of changing the parsers after every refactoring that changes the order/frequency of output.